content/en/getting_started/workflow_automation/_index.md

```diff
@@ -0,0 +1,155 @@
+8. Save your workflow.
+
+{{< img src="/getting_started/workflow_automation/trigger.png" alt="Trigger for Workflows">}}
+
+### Add the Jira and Slack actions
+To add the Jira step:
+1. On the workflow canvas, click **Add a step to get started**. 
+2. Search for the Jira action and select **Create issue**. Click the **Create issue** step and select your **Jira account**. This should correspond with the Jira URL found in the **Accounts** section of the Jira integration tile. 
+3. Enter the **Project** and **Issue type** for the Jira issue that the workflow creates. 
+4. Enter a **Summary** and **Description** for the Jira issue, using context variables to pass in data from the monitor that triggers the workflow. You can access a context variable in a step by enclosing it in double braces (`{{`). This following example uses the source, trigger and workflow variables to pass along the source that triggered the monitor alert, a link to the affected monitor, the workflow name, and the workflow ID:
+
+```
+The CPU usage is above the 95% threshold for {{ Trigger.hostname }}
+
+Please investigate - see this Datadog Dashboard to view all workflow executions: 
+https://app.datadoghq.com/dash/integration/workflow_automation?refresh_mode=sliding&from_ts=1698164453793&to_ts=1698168053793&live=true. 
+
+The workflow that created this Jira issue was 
+{{ WorkflowName }} : {{ WorkflowId }}
+
+The monitor that triggered the workflow can be found here: {{ Source.url }}
+```
+
+For more information on context variables, see **[Context variables][6]**. 
+
+5. You can test this Jira action by clicking the green **Test** icon in the **Configure** tab of the action.
+6. Save your workflow.
+
+You've set up a step to create a Jira ticket. Next, add the Slack step:
+1. Click the plus icon on the workflow canvas to add another step.
+2. Search for the Slack action and select **Send message**.  
+3. Enter the **Slack Workspace name**. 
+4. Enter the **Slack Channel name**.
+5. For a more useful Slack notification, you can use step output variables. Step output variables allow you to pass data from the Jira step to the Slack step in your workflow. Use the following message text to include the Jira issue key and Jira issue in the Slack message:
+
+```
+The monitor named, {{ Source.monitor.name }}, was triggered and created a new Jira issue 
+{{ Steps.Create_issue.issueKey }}: {{ Steps.Create_issue.issueUrl }}
+
+The workflow that created this Jira issue was {{ WorkflowName }}
+```
+
+6. You can test this Slack action by clicking the green **Test** icon in the **Configure** tab of the action.
+7. To view your workflow's name in the monitor's notification dropdown, save and publish your workflow. To publish the workflow, click **Publish** from the workflow's page.
```

Comment:
```suggestion
7. To view your workflow's name in the monitor's notification dropdown, save and publish your workflow. Click **Publish** from the workflow's page.
```
Is it easy for the user to navigate to the workflow page after testing the Slack action?